packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/positions/bringStateFineToAfterBatt.yml

```diff
@@ -1,6 +1,6 @@
 languageId: typescript
 command:
-  spokenForm: bring state fine to after bat
+  spokenForm: bring state fine after state bat
```

Comment:
This is correct, but I'm very curious where the original spoken form came from 😅

Reply:
Yeah there are a few of those where this original spoken form is just weird